packages/editor-ui/src/plugins/codemirror/completions/nativesAutocompleteDocs/luxon.static.docs.ts

```diff
@@ -192,46 +349,74 @@ export const luxonStaticDocs: Required<NativeDoc> = {
 				name: 'fromString',
 				docURL: 'https://moment.github.io/luxon/api-docs/index.html#datetimefromstring',
 				returnType: 'DateTime',
+				hidden: true,
 				args: [
 					{ name: 'text', type: 'string' },
 					{ name: 'fmt', type: 'string' },
-					{ name: 'options?', type: 'object' },
+					{ name: 'options', optional: true, type: 'Object' },
 				],
 			},
 		},
 		fromStringExplain: {
 			doc: {
 				name: 'fromStringExplain',
 				docURL: 'https://moment.github.io/luxon/api-docs/index.html#datetimefromstringexplain',
-				returnType: 'object',
+				returnType: 'Object',
+				hidden: true,
 				args: [
 					{ name: 'text', type: 'string' },
 					{ name: 'fmt', type: 'string' },
-					{ name: 'options?', type: 'object' },
+					{ name: 'options', optional: true, type: 'Object' },
 				],
 			},
 		},
 		max: {
 			doc: {
 				name: 'max',
+				description: i18n.baseText('codeNodeEditor.completer.luxon.dateTimeStaticMethods.max'),
 				docURL: 'https://moment.github.io/luxon/api-docs/index.html#datetimemax',
-				returnType: 'DateTime|undefined',
+				returnType: 'DateTime | undefined',
 				args: [
-					{ name: 'dateTime1', type: 'DateTime' },
-					{ name: '...' },
-					{ name: 'dateTimeN', type: 'DateTime' },
+					{
+						name: 'dateTimes',
+						variadic: true,
+						type: 'DateTime',
+						description: i18n.baseText(
+							'codeNodeEditor.completer.luxon.dateTimeStaticMethods.max.args.dateTimes',
+						),
+					},
+				],
+				examples: [
+					{
+						example:
+							"DateTime.max('2024-03-30T18:49'.toDateTime(), '2025-03-30T18:49'.toDateTime())",
+						evaluated: '2025-03-30T18:49',
+					},
 				],
 			},
 		},
 		min: {
 			doc: {
 				name: 'min',
+				description: i18n.baseText('codeNodeEditor.completer.luxon.dateTimeStaticMethods.min'),
 				docURL: 'https://moment.github.io/luxon/api-docs/index.html#datetimemin',
-				returnType: 'DateTime|undefined',
+				returnType: 'DateTime | undefined',
```

Comment:
If padding pipes with whitespace is the style, I think I've seen `string|string[]` elsewhere.

Reply:
I'll do a search, I prefer with whitespace